src/views/partials/singleJobCard.ejs

```diff
@@ -1,17 +1,35 @@
-<a href="<%=job.web_url%>" target="_blank">
-    <div
-      class="<%=job.status%>-card sm:p-5 sm:h-26 p-1 min-h-0 w-auto border-solid border-1 text-gray-400 bg-gray-50 transition duration-200 ease-in-out transform hover:scale-110">
-      <div class="relative">#<span class="<%=job.status%>-id">
-          <%= job.id %>
-        </span></div>
-      Created At: <span class="text-gray-400 relative">
-        <%= job.created_at %>
-      </span><br>
-      Started At: <span class="text-gray-400 relative">
-        <%= job.started_at %>
-      </span><br>
-      tags: <span class="text-gray-400 relative">
-        <%= job.tag_list %>
-      </span>
+  <a href="<%=job.web_url%>" target="_blank">
```

Comment:
I just realize there is a `jobCard.js` and `singleJobCard.js`. Can you help refactor and make them one single template file.